test/integration/api.bats

```diff
@@ -508,12 +569,39 @@ function teardown() {
 	skip
 }
 
-# FIXME
 @test "docker version" {
-	skip
+	start_docker 3
+	swarm_manage
+
+	# version
+	run docker_swarm version
+	[ "$status" -eq 0 ]
+	[ "${#lines[@]}" -ge 8 ]
+
+	# verify
+	client_reg='^Client version: [0-9]+\.[0-9]+\.[0-9]+$'
+	server_reg='^Server version: swarm/[0-9]+\.[0-9]+\.[0-9]+$'
+	[[ "${lines[0]}" =~ $client_reg ]]
+	[[ "${lines[5]}" =~ $server_reg ]]
 }
 
-# FIXME
 @test "docker wait" {
-	skip
+	start_docker 3
+	swarm_manage
+	
+	# run after 10 seconds, test_container will exit
+	run docker_swarm run -d --name test_container busybox sleep 10
+	[ "$status" -eq 0 ]
+
+	# make sure container exists and is up
+	run docker_swarm ps -l
+	[ "${#lines[@]}" -eq 2 ]
+	[[ "${lines[1]}" ==  *"test_container"* ]]
+	[[ "${lines[1]}" ==  *"Up"* ]]
+
+	# wait until exist(after 10 seconds)
+	run docker_swarm wait test_container
```

Comment:
The problem here is that if `wait` doesn't work, this will be stuck forever.

How about using the `timeout` tool?

```
run docker_swarm run -d --name test_container busybox sleep 3
[...]
run timeout 5 docker_swarm wait test_container
[ "$status" -eq 0 ]
```